packages/transit-vehicle-overlay/src/types.js

```diff
@@ -0,0 +1,24 @@
+// TODO: move to core-utils/types.js (maybe)
```

Comment:
This definitely seems like it belongs in core-utils/types. Also, I'd rename the type to transtiVehicleType.